src/applications/gi-sandbox/components/profile/EstimatedBenefits.jsx

```diff
@@ -92,25 +103,36 @@ export default function EstimatedBenefits({ profile, outputs, calculator }) {
 
       const learnMoreLink = `http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#${section.toLowerCase()}`;
       const headerId = `${_.snakeCase(title)}_header`;
+
       return (
         <div key={section} className="per-term-section">
-          <div className="link-header">
-            <span id={headerId}>
-              <strong>{title}</strong>
-            </span>
-            <span className="vads-u-padding-left--2">
-              (
-              <a
-                href={learnMoreLink}
-                target="_blank"
-                rel="noopener noreferrer"
-                aria-label={learnMoreAriaLabel || ''}
-              >
-                Learn more
-              </a>
-              )
-            </span>
-          </div>
+          {outputs.perTerm[section].title === 'Housing allowance' && isOJT ? (
```

Comment:
This text could change, the name of the `perTerm` property is `housingAllowance` and this should be checked for instead

Reply:
```suggestion
          {section === 'housingAllowance' && isOJT ? (
```

Don't need `outputs.perTerm[section]` here, properties were already plucked at start of map function

Reply:
Would be better in https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi-sandbox/selectors/calculator.js#L1201 to add a `perTerms.housingAllowance.modal` property with the value of `housingAllowanceOJT` and then check for the presence of the modal property, instead of coding this for a specific section

Reply:
if OJT for ☝️ comment add the `perTerms.housingAllowance.modal = 'housingAllowanceOJT';`